src/components/propositionalLogic/examplePage.tsx

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+
+import styles from "./propositionalLogic.module.scss";
+
+class ExamplesPropositionalLogic extends React.Component {
+	public render() {
+		return <div className={styles.site}>
+			<div className={styles.proofEditor}>
+				<p><button className={styles.buttons}
+					onClick={this.showElement}>
+					Beispiel 1
+						</button></p>
+				<table className={styles.toggleTable} id="toggleTable">
```

Comment:
Choosing an example via a button is valid. But it shouldn't be done via CSS when working in react. The I created just now doesn't contain a suggestion how to do this in react yet, but I can help you with this too. In the end it will make things a lot easier if understood.